examples/tutorials/first-figure.py

```diff
@@ -7,7 +7,8 @@
 
 .. note::
 
-    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    This tutorial assumes the use of a Python notebook, such as `IPython <https://www.pygmt.org/latest/install.html#using-pip>`__
```

Comment:
Is IPython a Python notebook? or is it notebook server/kernel?

Reply:
I was under the impression that it is a notebook. But I've only ever used Jupyter.

Reply:
If I understand it correctly, IPython is a console and also a kernel, and Jupyter Notebook use the IPython kernel to work.

Reply:
Do anyone just install IPython for PyGMT without Jupyter Notebook or JupyterLAB?

Reply:
We don't make any assumptions. I think it's common that people have ipython, but not jupyter notebook.